human_analysis/nbconverted/2_identify_generic_genes_pathways.py

```diff
@@ -83,25 +86,26 @@
 # In[4]:
 
 
-# Simulate multiple experiments
+"""# Simulate multiple experiments
 for i in range(num_runs):
-    generate_template_data.shift_template_experiment(
+    simulate_expression_data.shift_template_experiment(
         normalized_compendium_file,
         project_id,
+        metadata_col_id,
         NN_architecture,
         dataset_name,
         scaler,
         local_dir,
         base_dir,
-        i)
+        i)"""
 
 
 # Since this experiment contains both RNA-seq and smRNA-seq samples which are in different ranges so we will drop smRNA samples so that samples are within the same range. The analysis identifying these two subsets of samples can be found in this [notebook](../explore_data/0_explore_input_data.ipynb)
 
 # In[5]:
 
 
-# Remove subset of samples
+"""# Remove subset of samples
```

Comment:
Do these commented out sections still need to be in this notebook? If you want to remove them but refer back to them in the future if you need them again, you can just remove them - they'll still be in your commit history.

Generally I prefer to just remove things if they're no longer needed rather than commenting them out, since it can be hard to remember in the future why you commented something out or under what conditions you should uncomment it.

If this section of code should run only under certain conditions, it would be best to wrap it in a conditional, or at the very least add a comment describing when it should run.

Reply:
These are cells that were commented out just so that I could test the other cells. In this case the entire notebook should be run in its entirety and not pieces so I'll uncomment them all